### PR TITLE
OBaaS::Docs - Add prereqs and change Setup menu

### PR DESCRIPTION
**Pull request by pasimoes**
This PR addresses Issue #522 and changes the Setup menu. The usual sequence of steps are:
- setup platform 
- configure the local workstation
- get access to Kubernetes cluster and Database

### CODE REVIEW

File: docs-source/spring/content/development/cli/_index.md [hunk 1]
```diff
@@ -1,16 +1,10 @@
 ---
-title: Using the CLI
+title: Using the Oracle Spring CLI
```
Comment (markxnelson) [inline docs-source/spring/content/development/cli/_index.md:2]: you cannot call it "Oracle Spring CLI" - you need to use the correct product name, e.g. "CLI for Oracle Backend as a Service for Spring Cloud" 
i think the title "using the CLI" was fine as it was

File: docs-source/spring/content/development/cli/_index.md [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
 ---
 
-## Setup
+The Oracle Backend as a Service offer a Spring Cloud command-line tool, `oracle-spring`, to be use. The CLI commands simplify the deployment of applications containing microservices as well as bindings with the resources they use.
```
Comment (markxnelson) [inline docs-source/spring/content/development/cli/_index.md:5]: use correct product name. 
offers is plural
"to be use" means nothing - remove it
"they" -> "those applciations" 
"applications containing microservices" -> "microservices applications"

File: docs-source/spring/content/platform/_index.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+title: Platform
```
Comment (markxnelson) [inline docs-source/spring/content/platform/_index.md:2]: this is an unnecessary change - remove it

File: docs-source/spring/content/setup/_index.md [hunk 4]
```diff
@@ -36,13 +36,18 @@ Oracle Backend as a Service for Spring Cloud is available in the [OCI Marketplac
...
-
+- On Oracle Cloud:
+  - An OCI account in a tenancy with sufficient quota to create:
+  - An OCI Container Engine for Kubernetes cluster, plus a node pool with three worker nodes
+  - A VCN with at least two public IP's available
+  - A public load balancer
+  - An Oracle Autonomous Database - Shared instance
+  - At least one free OCI Auth Token (note that the maximum is two per user)
+
+- On Local Workstation:
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:47]: On your local workstation - don't capitalize

File: docs-source/spring/content/setup/_index.md [hunk 5]
```diff
@@ -121,3 +124,22 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
     <!-- spellchecker-disable -->
     {{< img name="oci-stack-apply-logs" size="large" lazy=false >}}
     <!-- spellchecker-enable -->
+
+## Setup the local workstation
+
+1. Install Local Tools
+
+    Before you set up the [Kubernetes access](./cluster-access), you must have already done the following:
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:132]: change the tense.  tell them what to do now, not what they needed to have already done

File: docs-source/spring/content/setup/_index.md [hunk 6]
```diff
@@ -121,3 +124,22 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
...
+
+    - generated an API signing key pair
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:134]: include links to docs to tell them how to do these things

File: docs-source/spring/content/setup/_index.md [hunk 7]
```diff
@@ -121,3 +124,22 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
...
+    - installed and configured the Oracle Cloud Infrastructure CLI (version 2.6.4 or later)
+    - installed and configured the Kubernetes command-line tool (kubectl)
+
+    To use `kubectl` on your local machine, you will need to have the [OCI CLI installed and configured](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm).
+
+2. Install Oracle Backend as a Service for Spring Cloud command-line
+
+    The Oracle Backend as a Service for Spring Cloud command-line tool, `oracle-spring`, is available for Linux and Mac systems. Download the binary you want from the [Releases](https://github.com/oracle/microservices-datadriven/releases/tag/OBAAS-1.0.0) page and add it to your PATH environment variable.  You may like to rename the binary to remove the suffix.
+
+    If you're environment is a Linux or Mac machine you need to execute `chmod +x` on the downloaded binary. Also if your environment is a Mac you need execute the following command `sudo xattr -r -d com.apple.quarantine <downloaded-file>` otherwise will you get a security warning and the CLI will not work.
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:145]: your, not you're
don't say "you need to execute" just tell them to do it, and don't use the word "execute" 

You can just say "On MacOS, run this command to (do whatever)" etc

File: docs-source/spring/content/setup/cluster-access/_index.md [hunk 8]
```diff
@@ -17,20 +23,16 @@ kubeconfig_cmd = "oci ce cluster create-kubeconfig
...
 ```
 
-To use `kubectl` on your local machine, you will need to have the [OCI CLI installed and configured](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm).
-
-For more information about working with the Kubernetes cluster, see [Setting Up Cluster Access](https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengdownloadkubeconfigfile.htm#localdownload) in the OCI documentation.
-
 **Note:** The generated `kubeconfig` file will only work if you are using the `DEFAULT` profile in your OCI CLI configuration file. If you are using a different OCI CLI profile you must add `--profile <NAME>` to the command by editing the generated Kubernetes config file and add the following lines:
 
-```
+```yaml
```
Comment (markxnelson) [inline docs-source/spring/content/setup/cluster-access/_index.md:28]: this is not yaml

File: docs-source/spring/data/menu/main.yaml [hunk 9]
```diff
@@ -1,6 +1,11 @@
 main:
   - name: Setup
     ref: "/setup"
+    sub:
```
Comment (markxnelson) [inline docs-source/spring/data/menu/main.yaml:4]: do not move these. put them back where they were so that 1) it is consistent with mbaas docs, and 2) database is not hidden away in the weeds

File: docs-source/spring/data/menu/main.yaml [hunk 10]
```diff
@@ -21,7 +26,7 @@ main:
         ref: "/platform/conductor"
       - name: Spring Admin
         ref: "/platform/spring-admin"
```
Comment (markxnelson) [inline docs-source/spring/data/menu/main.yaml:28]: you cannot say "Oracle Spring (anything)" - not our trademark

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: docs-source/spring/content/development/cli/_index.md (hunk 1 above)
Comment (pasimoes) [reply docs-source/spring/content/development/cli/_index.md:2]: `CLI for Oracle Backend as a Service for Spring Cloud` is good enough to a description but not for a title or menu and only CLI is not meaningful. OBaaS CLI?

Comment (markxnelson) [reply docs-source/spring/content/development/cli/_index.md:2]: "Using the CLI" is fine - it is obvious from the context what we are talking about

File: docs-source/spring/content/setup/cluster-access/_index.md (hunk 8 above)
Comment (pasimoes) [reply docs-source/spring/content/setup/cluster-access/_index.md:28]: kubeconfig is a yaml file

File: docs-source/spring/content/development/cli/_index.md [hunk 11]
```diff
@@ -1,16 +1,10 @@
 ---
-title: Using the CLI
+title: Using the OBaaS CLI
 ---
 
-## Setup
+The Oracle Backend as a Service for Spring Cloud offers a command-line tool, `oracle-spring`. The CLI commands simplify the deployment of microservices applications as well as bindings with the resources they use.
```
Comment (markxnelson) [inline docs-source/spring/content/development/cli/_index.md:5]: its `obaas` not `oracle-spring` - we renamed the binary

**Review by markxnelson — CHANGES_REQUESTED** [review]
couple of small changes needed - i think  you need to rebase too, i think paul updated some of the stuff you are also proposing to touch here

File: docs-source/spring/content/development/cli/_index.md [hunk 12]
```diff
@@ -20,16 +14,14 @@ If you're environment is a Linux or Mac machine you need to execute `chmod +x` o
 
 2. Start the CLI in interactive mode by simply running `oracle-spring` from your terminal window.
 
-```shell
-    oracle-spring
-```
+    ```shell
+        oracle-spring
```
Comment (markxnelson) [inline docs-source/spring/content/development/cli/_index.md:18]: obaas

File: docs-source/spring/content/setup/_index.md [hunk 13]
```diff
@@ -125,3 +127,34 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
...
+          The key is uploaded and its fingerprint is displayed (for example, d1:b2:32:53:d3:5f:cf:68:2d:6f:8b:5f:77:8f:07:13).
+
+    - install and configure the Oracle Cloud Infrastructure CLI
+        1. Install the Oracle Cloud Infrastructure CLI version 2.6.4 (or later). See [Quickstart](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm#Quickstart).
+        2. Configure the Oracle Cloud Infrastructure CLI. See [Configuring the CLI](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliconfigure.htm#Configuring_the_CLI).
+
+
+2. Install Oracle Backend as a Service for Spring Cloud command-line
+
+    The **Oracle Backend as a Service for Spring Cloud** command-line tool, `oracle-spring`, is available for Linux and Mac systems. Download the binary you want from the [Releases](https://github.com/oracle/microservices-datadriven/releases/tag/OBAAS-1.0.0) page and add it to your PATH environment variable.  You may like to rename the binary to remove the suffix.
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:158]: obaas

File: docs-source/spring/content/setup/_index.md [hunk 14]
```diff
@@ -125,3 +127,34 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
...
+
+    If you're environment is a Linux or Mac machine you need to execute `chmod +x` on the downloaded binary. Also if your environment is a Mac you need execute the following command `sudo xattr -r -d com.apple.quarantine <downloaded-file>` otherwise will you get a security warning and the CLI will not work.
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:160]: you need to rebase, this is already updated

File: docs-source/spring/data/menu/main.yaml [hunk 15]
```diff
@@ -33,8 +38,8 @@ main:
   - name: Security
     ref: "/security"
   - name: Database Access
-    ref: "/database"
+    ref: "/setup/database"
```
Comment (markxnelson) [inline docs-source/spring/data/menu/main.yaml:41]: no spurious or unnecessary moving of content please

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: docs-source/spring/content/setup/_index.md (hunk 14 above)
Comment (pasimoes) [reply docs-source/spring/content/setup/_index.md:160]: this is coming from another PR not merged yet.

File: docs-source/spring/content/setup/_index.md [hunk 16]
```diff
@@ -125,3 +127,34 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
...
+
+        1. Use OpenSSL commands to generate the key pair in the required PEM format. If you're using Windows, you'll need to install Git Bash for Windows and run the commands with that tool. See [How to Generate an API Signing Key](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#two).
+        2. Copy the contents of the public key to the clipboard (you'll need to paste the value into the Console later).
+
+    - add the public key value of the API signing key pair to the User Settings for your username
+       1. In the top-right corner of the Oracle Cloud Infrastructure Console, open the **Profile** menu (User menu icon) and then click **User Settings** to view the details.
+       2. Click **Add Public Key**.
+       3. Paste the public key's value into the window and click **Add**.
+
+          The key is uploaded and its fingerprint is displayed (for example, d1:b2:32:53:d3:5f:cf:68:2d:6f:8b:5f:77:8f:07:13).
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:149]: i assume this is a made-up fingerprint?

File: docs-source/spring/content/setup/_index.md (hunk 7 above)
Comment (markxnelson) [reply docs-source/spring/content/setup/_index.md:160]: still has incorrect word "you're" and unacceptable word "execute"

**Review by markxnelson — CHANGES_REQUESTED** [review]

File: docs-source/spring/content/setup/_index.md (hunk 16 above)
Comment (pasimoes) [reply docs-source/spring/content/setup/_index.md:149]: yes.. it was copied from official docs.
https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengdownloadkubeconfigfile.htm#

File: docs-source/spring/content/setup/_index.md (hunk 7 above)
Comment (pasimoes) [reply docs-source/spring/content/setup/_index.md:160]: replaced all occurrences when possible.

**Review by markxnelson — APPROVED** [review]
